dependencies.gradle

```diff
@@ -33,7 +33,7 @@ def vanniktechEmoji = "0.15.0"
 def fragment = "1.5.2"
 
 // Testing
-def mockk = "1.12.3" // We need to use 1.12.3 to have mocking in androidTest until a new version is released: https://github.com/mockk/mockk/issues/819
+def mockk = "1.12.7" // We need to use 1.12.3 to have mocking in androidTest until a new version is released: https://github.com/mockk/mockk/issues/819
```

Comment:
Seems to be OK since 1.12.5, https://github.com/mockk/mockk/issues/819#issuecomment-1196961428